content/en/docs/instrumentation/net/netframework.md

```diff
@@ -44,11 +44,11 @@ using OpenTelemetry.Trace;
 
 public class WebApiApplication : HttpApplication
 {
-    private readonly TracerProvider _tracerProvider;
+    private TracerProvider _tracerProvider;
 
     protected void Application_Start()
     {
-        this.tracerProvider = Sdk.CreateTracerProviderBuilder()
+        _tracerProvider = Sdk.CreateTracerProviderBuilder()
```

Comment:
why remove this.? 
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/examples/AspNet/Global.asax.cs#L69

Reply:
It's an IDE0003 - the `this` is considered redundant by Visual Studio.